tests/plugins/widget/manual/contextmenushortcuts.html

```diff
@@ -0,0 +1,10 @@
+<div id="editor">
+	<p><span class="math-tex">\(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\)</span></p>
```

Comment:
If you decide to replace `mathjax` with `placeholder`, you can use double brackets to upcast text into a widget. E.g. `[[focus me]]`.